# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/fcft/pixman.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- fcft/fcft/fcft.h.vanilla	2022-08-22 22:08:31.514291890 +0200
+++ fcft/fcft/fcft.h	2022-08-22 22:08:45.842293040 +0200
@@ -4,7 +4,7 @@
 #include <stdint.h>
 #include <stddef.h>
 
-#include <pixman.h>
+#include <pixman-1/pixman.h>
 
 /*
  * Logging
